src/components/List.jsx

```diff
@@ -8,6 +14,26 @@ export default function List() {
   const [docs, setDocs] = useState([]);
   const [userToken] = useState(getUser());
 
+  const handleCheckBox = (e, item) => {
+    e.preventDefault();
+    let now = Timestamp.now().seconds;
+    const docItem = doc(db, userToken, item.id);
+    updateDoc(docItem, {
+      lastPurchaseDate: now,
+    });
+  };
+
+  const check24Hrs = (item) => {
+    let now = Timestamp.now().seconds;
+    let itemPurchaseDate = item.data().lastPurchaseDate;
+    let difference = now - itemPurchaseDate;
+    // We are using 86400 which is 24 hours in seconds since `now` is also in seconds
```

Comment:
Nice call-out! 'Magic numbers' like this can be challenging to work with. Another way folks deal with magic numbers is by setting them up as constants e.g. `const secondsInDay = 86400` then using this in your functions e.g. `difference < secondsInDay`.

Reply:
Totally makes sense! Added a change.